source/refunding_payments/index.html.md.erb

```diff
@@ -182,70 +180,71 @@ For example, the response body for a refund would include:
 target="blank">Find payment refund by ID</a> (link opens in new window) to
-check the processing status of the refund until it changes to either
-``success`` or ``error``.
+check the processing status of the refund, until it changes to either
+`success` or `error`.
 
 It will typically take 30 minutes for the status to change. You should not
 check the status more than once every 5 minutes.
 
 In the event of an error, GOV.UK Pay will not currently provide any more
 information. Please [contact
-us](/support_contact_and_more_information/#contact-us) if more information is
-required about why a refund failed.
+us](/support_contact_and_more_information/#contact-us) if you need more
+information required about why a refund failed.
 
-## Refunding from the GOV.UK Pay admin site
+## Refunding from the GOV.UK Pay admin tool
 
 You can use the [GOV.UK Pay admin
 site](https://selfservice.payments.service.gov.uk) to view transactions and
 issue refunds.
 
-Go to the __Transactions__ section of your accoun page to see a list of
-transactions. For example:
+Go to the __Transactions__ section to see a list of transactions. For example,
+in the following image::
 
 ![](images/transaction+list+image+4.png)
 
 In this list, you can select an individual payment to see details of that
 payment, including any previous refunds.
 
 For a successful payment with no previous refunds, you can use the red
-**Refund payment** button to carry out a full or partial payment.
+__Refund payment__ button to make a full or partial payment.
 
 ![](images/refund-payment-button.png)
 
-## Refund notifications
+## Refund email notifications
+
+You can use the GOV.UK Pay admin tool to [send email notifications to
+users](https://selfservice.payments.service.gov.uk/email-notifications).
 
-End users are automatically notified by email about successful payments.  This
-depends on settings you have entered in the self-service admin site. Users are
-not notified when a payment has either failed or is refunded - either manually
-or via the API. You should arrange to notify end users about payment failures
-and refunds as appropriate for your service.
+Users are not notified when a payment has either failed or is refunded. You
+should arrange to notify end users about payment failures and refunds as
+appropriate for your service.
 
 ## Refund destination
 
 Refunds are sent back to the source account of the original payment. It is not
 possible to refund a payment to another card or bank account. If the original
-card expired or was canceled, the bank will attempt to credit the customer’s
+card expired or was cancelled, the bank will attempt to credit the customer’s
 new card with the refund. If this attempt fails, or the customer does not have
-a new card, the refund will have to be made using another channel (eg bank
-transfer or cheque).
+a new card, the refund will have to be made using another method. For example,
+bank transfer or cheque.
```

Comment:
I think we should make it clearer here that it's the service's responsibility to use a different method to refund - GOV.UK Pay doesn't do it, nor does the bank.